clar2wasm/src/words/equal.rs

```diff
@@ -0,0 +1,55 @@
+use crate::wasm_generator::{clar2wasm_ty, ArgumentsExt, GeneratorError, WasmGenerator};
+use clarity::vm::{ClarityName, SymbolicExpression, types::TypeSignature, types::SequenceSubtype, types::StringSubtype};
+use walrus::ValType;
+
+use super::Word;
+
+#[derive(Debug)]
+pub struct Equal;
+
+impl Word for Equal {
+    fn name(&self) -> ClarityName {
+        "is-eq".into()
+    }
+
+    fn traverse(
+        &self,
+        generator: &mut WasmGenerator,
+        builder: &mut walrus::InstrSeqBuilder,
+        _expr: &SymbolicExpression,
+        args: &[SymbolicExpression],
+    ) -> Result<(), GeneratorError> {
+        let mut results = Vec::new();
+
+        // Traverse the first two values, leaving them on the stack
+        let first_op = args.get_expr(0)?;
+        generator.traverse_expr(builder, first_op)?;
+        let sec_op = args.get_expr(1)?;
+        generator.traverse_expr(builder, sec_op)?;
```

Comment:
You'll never believe it, but `is-eq` can have only one argument (in this case the end answer is always true)